paymentsheet/src/main/java/com/stripe/android/customersheet/injection/CustomerSheetViewModelModule.kt

```diff
@@ -145,20 +146,24 @@ internal interface CustomerSheetViewModelModule {
             savedStateHandle: SavedStateHandle,
             paymentConfigurationProvider: Provider<PaymentConfiguration>,
             bacsMandateConfirmationLauncherFactory: BacsMandateConfirmationLauncherFactory,
+            googlePayPaymentMethodLauncherFactory: GooglePayPaymentMethodLauncherFactory,
             stripePaymentLauncherAssistedFactory: StripePaymentLauncherAssistedFactory,
             statusBarColor: Int?,
             intentConfirmationInterceptor: IntentConfirmationInterceptor,
             errorReporter: ErrorReporter,
+            logger: Logger,
         ): IntentConfirmationHandler.Factory {
             return IntentConfirmationHandler.Factory(
                 intentConfirmationInterceptor = intentConfirmationInterceptor,
                 paymentConfigurationProvider = paymentConfigurationProvider,
                 stripePaymentLauncherAssistedFactory = stripePaymentLauncherAssistedFactory,
+                googlePayPaymentMethodLauncherFactory = googlePayPaymentMethodLauncherFactory,
```

Comment:
does customer sheet support google pay? I would've thought no, based on the fact that we pass a null configuration to `toPaymentConfirmationOption` in `CustomerSheetViewModel`. If that's the case, can we make the google pay payment method launcher factory nullable here or something?